segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBCache.java

```diff
@@ -255,6 +259,36 @@ private Options createDatabaseOptions() {
                 .setTableFormatConfig(tableFormatConfig)
                 .setOptimizeFiltersForHits(true)
                 .setUseDirectReads(true);
+
+        InfoLogLevel logLevel = translateRocksDBLogLevel(rocksDBLogLevel);
+        Options logOptions = new Options().
+                setInfoLogLevel(logLevel).
+                setCreateIfMissing(true);
+
+        Logger logger = new Logger(logOptions) {
+            @Override
+            protected void log(InfoLogLevel infoLogLevel, String logMsg) {
+                switch (infoLogLevel) {
+                    case DEBUG_LEVEL:
+                        log.debug(logMsg);
```

Comment:
This is good, but I want more precision in these messages. This will log a generic message coming from `RocksDBCache`, however we have multiple instances of this class (one per Segment Container). Each such class has an `id` (which contains `Cache_<container-id>`). It would be very helpful in debugging to figure out where this came from.

Please change these `log.xyz` calls to something like `log.xyz("{}: {}", this.id, logMsg)` (`xyz` is the method you call on `log`).

Reply:
Understood! will modify that.